### PR TITLE
feat: add deployer account and deploy fee to madara genesis

### PR DESCRIPTION
**Pull request by ftupas**
Time spent on this PR: 0.1

Resolves: #436 

# Pull Request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Testing

# What is the new behavior?

Deployer account and deploy fee added to madara genesis

# Does this introduce a breaking change?

- [ ] Yes
- [X] No

# Added changes to [`CHANGELOG.md`](../CHANGELOG.md)

- [X] Yes
- [ ] No

### CODE REVIEW

File: crates/hive-utils/src/hive/genesis.rs [hunk 1]
```diff
@@ -43,6 +43,7 @@ impl HiveGenesisConfig {
 lazy_static! {
     pub static ref KAKAROT_ADDRESSES: FieldElement = FieldElement::from_hex_be("0x9001").unwrap(); // Safe unwrap, 0x9001
     pub static ref BLOCKHASH_REGISTRY_ADDRESS: FieldElement = FieldElement::from_hex_be("0x9002").unwrap(); // Safe unwrap, 0x9002
+    pub static ref DEPLOYER_ACCOUNT_ADDRESS: FieldElement = FieldElement::from_hex_be("0x9003").unwrap(); // Safe unwrap, 0x9003
```
Comment (Eikix) [inline crates/hive-utils/src/hive/genesis.rs:46]: can this change in another place and create a desync? or is it fixed forever

File: crates/hive-utils/src/hive/genesis.rs [hunk 2]
```diff
@@ -107,6 +112,8 @@ pub async fn serialize_hive_to_madara_genesis_config(
         ("externally_owned_account", eoa_class_hash),
         ("account_proxy_class_hash", account_proxy_class_hash),
         ("blockhash_registry_address", *BLOCKHASH_REGISTRY_ADDRESS),
+        // TODO: Use DEPLOY_FEE constant https://github.com/kkrt-labs/kakarot-rpc/pull/431/files#diff-88f745498d0aaf0b185085d99a74f0feaf253f047babc85770847931e7f726c3R125
+        ("deploy_fee", FieldElement::from(100000_u64)),
```
Comment (Eikix) [inline crates/hive-utils/src/hive/genesis.rs:116]: @bajpai244 will this become a problem? since it's not supposed to be a constant

**Review by Eikix — COMMENTED** [review]
lgtm

File: crates/hive-utils/src/hive/genesis.rs (hunk 1 above)
Comment (ftupas) [reply crates/hive-utils/src/hive/genesis.rs:46]: If we initialize Madara with this config, these contracts will be fixed to these addresses

File: Cargo.toml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 [workspace]
-members = ["crates/eth-rpc", "crates/core", "crates/conformance-test-utils"]
+members = ["crates/eth-rpc", "crates/core", "crates/hive-utils"]
```
Comment (ftupas) [inline Cargo.toml:2]: As discussed with @greged93 , renaming this crate so it's more descriptive

File: crates/hive-utils/src/hive/genesis.rs (hunk 1 above)
Comment (Eikix) [reply crates/hive-utils/src/hive/genesis.rs:46]: alright

File: crates/hive-utils/src/hive/genesis.rs (hunk 2 above)
Comment (bajpai244) [reply crates/hive-utils/src/hive/genesis.rs:131]: So, this value can always be updated later on, u have to firstly initialize Kakarot with one `deploy_fee` which seems to be the case here, and then an off-chain service can keep updating this value, so technically we are not making this a constant here, we are just initializing Kakarot.

**Review by Eikix — APPROVED** [review]
lgtm

**Pull request merged**